src/SprykerEco/Zed/Adyen/Business/Hook/Saver/MakePayment/CreditCardSaver.php

```diff
@@ -38,6 +43,8 @@ protected function updatePaymentAdyenTransfer(
             $paymentAdyenTransfer->setPaymentData($response->getMakePaymentResponse()->getPaymentData());
         }
 
+        $this->adyenApiResponseTransfer = $response;
```

Comment:
It's not a good practice to use class property to store some data. Could you please check if it's possible to refactor `AbstractSaver` instead somehow. Probably you can use `AdyenApiResponseTransfer` to store there needed payment status.